src/robotFactory.js

```diff
@@ -1,10 +1,67 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name, weight, {
+    x = 0, y = 0,
+  }, chipVersion) {
```

Comment:
Якщо переносиш аргументи тоді кожен повинен бути з нового рядка
```suggestion
  constructor(
    name,
    weight,
    {
      x = 0,
      y = 0,
    },
   chipVersion,
) {
```